src/Pages/Login/LoginInput.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import LoginLinkBox from './LoginLinkBox';
+
+export default class LoginInput extends Component {
+  state = {
+    isPasswordSectetOnOff: false,
+  };
+
+  handleSecret = e => {
+    const { isPasswordSectetOnOff } = this.state;
+    this.setState({
+      isPasswordSectetOnOff: !isPasswordSectetOnOff,
+    });
+  };
+
+  handleBlur = e => {};
+
+  render() {
+    const { isPasswordSectetOnOff } = this.state;
+
+    const {
+      name,
+      handleChange,
+      warnText,
+      placeholder,
+      type,
+      linkText,
+      subtext,
+      value,
+      inputValue,
+    } = this.props;
+    return (
+      <div>
+        <input
+          inputValue={inputValue}
+          name={name}
+          className="loginInput"
+          placeholder={placeholder}
+          type={type}
+          value={value}
+          onChange={e => handleChange(e)}
+        />
+
+        {inputValue.length === 0 ? (
+          <div className="loginWarnTextTrue">{warnText}</div>
+        ) : (
+          <div className="loginWarnText">{warnText}</div>
+        )}
+
+        <span
+          className={inputValue.length > 0 ? 'passwordEyeFalse' : 'passwordEye'}
+          onClick={this.handleSecret}
+        >
+          {isPasswordSectetOnOff ? (
+            <i class="far fa-eye-slash"></i>
+          ) : (
+            <i class="far fa-eye"></i>
+          )}
```

Comment:
```suggestion
        <div className={`loginWarnText ${inputValue.length ? '' : 'off'}`}>{warnText}</div>
        <span
          className={`passwordEye ${inputValue.length ? 'off' : ''}`}
          onClick={this.handleSecret}
        >
        <i className={`far fa-eye ${isPasswordSectetOnOff ? '-slash' : ''}` />
```
- 일관성 및 간결함을 위해 위와 같이 템플릿 리터럴 사용하여 작성하는게 어떨까요?
- react에서는 class가 아니라 className 입니다. 콘솔 창에 에러 뜨는거 확인하시고 꼭 제거해주세요. 나중에 build 했을 때 fail하게 됩니다.